app/design/adminhtml/default/default/template/sales/order/view/info.phtml

```diff
@@ -19,21 +19,16 @@
 <?php
 $orderAdminDate = $this->formatDate($_order->getCreatedAtDate(), 'medium', true);
 $orderStoreDate = $this->formatDate($_order->getCreatedAtStoreDate(), 'medium', true);
+$emailSentInfo  = $_order->getEmailSent() ? Mage::helper('sales')->__('Email sent') : Mage::helper('sales')->__('Email not sent');
```

Comment:
Should it be "Sent" and "Not Sent"?

Reply:
Thanks for suggestion, but please use https://egghead.io/lessons/github-add-suggestions-in-a-github-pr-review